client/src/components/end-screen/end-screen.vue

```diff
@@ -30,6 +30,19 @@ const earnings = computed(() => {
   if (!initialBalance || !balance) return new BigNumber(0);
   return balance.minus(initialBalance);
 });
+
+const insuffientBalance = computed(() => {
```

Comment:
proposing booleans to use a prefix of `is`, `should` or `has` for example

`isBalanceInsufficient` or `hasInsufficientBalance`

```suggestion
const isBalanceInsufficient = computed(() => {
```